### ISSUE TITLE
The Entmax-alpha and Entmax15 Feature in contrib

### ISSUE THREAD

**Issue by rwbfd** [pytorch/pytorch#49680]
## 🚀 Feature

Realization of Entmax-$\alpha$ as described in [(https://arxiv.org/abs/1905.05702)] in contrib.

## Motivation

The Entmax-$\alpha$ is one of the most promising approach that leads to a relatively sparse softmax version. With $\alpha=1$, it boils down to softmax, and with $\alpha=2$ it boils down to sparsemax. 

I have already implement it. However, I wonder whether this has any chance of being merged. 

cc @albanD @mruberry @jbschlosser

**Comment by ngimel** [comment]
Merging features into pytorch core requires wide adoption. You are welcome to implement as an add-on, it will be helpful if you provide a link to it in this issue, so that people are able to find it.

**Comment by rwbfd** [comment]
@ngimel Just a stupid question. By add-on you mean in contrib? Or another repo? This is my first time trying to contribute to torch eco-system so I am not exactly sure.

**Comment by ngimel** [comment]
Another repo. Popular third-party projects get included in pytorch ecosystem https://pytorch.org/ecosystem/

**Comment by rwbfd** [comment]
@ngimel OK, here it is! https://github.com/rwbfd/pytorch_addon.git. Hope it will grow! I will close the issue now.
